CHANGELOG.rst

```diff
@@ -20,6 +20,7 @@ Removed
 =======
 - Removed dependency from `amlight/scheduler`
 - Removed support for OpenFlow 1.0
+- Unsubscribe to the `amlight/flow_stats.flows_updated` event
```

Comment:
@gretelliz please add more line stating that this NApp no longer depends on `flow_stats`. Also, update it accordingly on the `kytos.json` file.

Reply:
Done!